intermediate/indexing/advanced-indexing.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>Would be good to expand this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done! The updated learning objectives:</p><p> </p><p>## Learning Objectives</p><p> </p><p>* Orthogonal vs. Pointwise (Vectorized) Indexing.</p><p>* Pointwise indexing in Xarray to extract data at a collection of points.</p><p>* Understand the difference between NumPy and Xarray indexing behavior.</p>